.appveyor.yml

```diff
@@ -0,0 +1,74 @@
+environment:
+#  nodejs_version: "0.12.7"
+  matrix:
+    # First group is 32 bit python 3.4
```

Comment:
Ideally we could also add 32 bit python and pyqt to provide 32 binaries for linux. But it should probably be removed until such a feature is implemented.